stack-overflow/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './App.css';
+import SmartLoginView from './view/SmartLoginView';
+import SmartQuestionView from "./view/SmartQuestionView";
+import {HashRouter, Switch, Route} from "react-router-dom";
+
+class App extends Component {
+
+  
+  render() {
+    return (
+      <div className="App">
+        <HashRouter>
+          <Switch>
+            <Route exact={true} component={SmartLoginView} path="/" />
+            <Route exact={true} component={SmartQuestionView} path="/logged-user" />
```

Comment:
The path for the question view is a little missleading. I would call it simply `/questions` instead. You will most likely have several paths which are accessible only by logged in users.